std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -142,6 +142,20 @@ public static Column fromItems(String name, List<Value> items) {
     return new Column(name, storage);
   }
 
+  /** Creates a column from an Enso array. No polyglot conversion happens. This is unsafe */
+  public static Column fromItemsRaw(String name, List<Object> items, StorageType expectedType) throws ClassCastException {
```

Comment:
At this point, I don't think we should add this to the core `Column` type.
It can be a static method anywhere from my read so I suggest it goes into the exploratory helper Java.

Reply:
I wanted to keep it in std-lib, because I actually plan to use it in #6111 - there when we know the expected value type, if the value type is _not_ `Mixed` nor `Date` (etc.), we can _guarantee_ no dates will come in - and so we do not need the conversion to happen. This will make the map operation and construction of columns like integer column much faster.

I can make it a static, but since I plan to use it in the main codebase soon, I did not want to move it to the helpers - since I'd have to move it back again when implementing #6111